histogramvec/histogramvec.go

```diff
@@ -87,7 +87,17 @@ func (hv *HistogramVec) Ensure(labels []string) {
 	}
 }
 
-// Histograms returns the currently managed Histograms.
+// Histograms returns a copy of the currently managed Histograms.
 func (hv *HistogramVec) Histograms() map[string]*histogram.Histogram {
```

Comment:
Isn't that a problem we can solve it by value receiver?

Reply:
No, even if we would use value receiver, we would still have to copy the pointer within the histogram.